RecoJets/JetProducers/python/PileupJetIDCutParams_cfi.py

```diff
@@ -71,450 +71,3 @@
-
-    #Medium Id
-    Pt010_Medium   = cms.vdouble(-999.,-999.,-999.,-999.),
-    Pt1020_Medium  = cms.vdouble(-999.,-999.,-999.,-999.),
-    Pt2030_Medium  = cms.vdouble(-999.,-999.,-999.,-999.),
-    Pt3040_Medium  = cms.vdouble(-999.,-999.,-999.,-999.),
-    Pt4050_Medium  = cms.vdouble(-999.,-999.,-999.,-999.),
-
-    #Loose Id
-    Pt010_Loose    = cms.vdouble(-999.,-999.,-999.,-999.),
-    Pt1020_Loose   = cms.vdouble(-999.,-999.,-999.,-999.),
-    Pt2030_Loose   = cms.vdouble(-999.,-999.,-999.,-999.),
-    Pt3040_Loose   = cms.vdouble(-999.,-999.,-999.,-999.),
-    Pt4050_Loose   = cms.vdouble(-999.,-999.,-999.,-999.)
-)
-
-
-PuJetIdCutBased_wp = cms.PSet(
-    #4 Eta Categories  0-2.5 2.5-2.75 2.75-3.0 3.0-5.0
-    #betaStarClassic/log(nvtx-0.64) Values
-    #Tight Id
-    Pt010_BetaStarTight    = cms.vdouble( 0.15, 0.15, 999., 999.),
-    Pt1020_BetaStarTight   = cms.vdouble( 0.15, 0.15, 999., 999.),
-    Pt2030_BetaStarTight   = cms.vdouble( 0.15, 0.15, 999., 999.),
-    Pt3040_BetaStarTight   = cms.vdouble( 0.15, 0.15, 999., 999.),
-    Pt4050_BetaStarTight   = cms.vdouble( 0.15, 0.15, 999., 999.),
-    
-    #Medium Id => Daniele
-    Pt010_BetaStarMedium   = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt1020_BetaStarMedium  = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt2030_BetaStarMedium  = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt3040_BetaStarMedium  = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt4050_BetaStarMedium  = cms.vdouble( 0.2, 0.3, 999., 999.),
-
-    #Loose Id
-    Pt010_BetaStarLoose    = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt1020_BetaStarLoose   = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt2030_BetaStarLoose   = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt3040_BetaStarLoose   = cms.vdouble( 0.2, 0.3, 999., 999.),
-    Pt4050_BetaStarLoose   = cms.vdouble( 0.2, 0.3, 999., 999.),
-
-    #RMS variable
-    #Tight Id
-    Pt010_RMSTight         = cms.vdouble( 0.06, 0.07, 0.04, 0.05),
-    Pt1020_RMSTight        = cms.vdouble( 0.06, 0.07, 0.04, 0.05),
-    Pt2030_RMSTight        = cms.vdouble( 0.05, 0.07, 0.03, 0.045),
-    Pt3040_RMSTight        = cms.vdouble( 0.05, 0.06, 0.03, 0.04),
-    Pt4050_RMSTight        = cms.vdouble( 0.05, 0.06, 0.03, 0.04),
-    
-    #Medium Id => Daniele
-    Pt010_RMSMedium        = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-    Pt1020_RMSMedium       = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-    Pt2030_RMSMedium       = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-    Pt3040_RMSMedium       = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-    Pt4050_RMSMedium       = cms.vdouble( 0.06, 0.03, 0.03, 0.04),
-
-    #Loose Id
-    Pt010_RMSLoose         = cms.vdouble( 0.06, 0.05, 0.05, 0.07),
-    Pt1020_RMSLoose        = cms.vdouble( 0.06, 0.05, 0.05, 0.07),
-    Pt2030_RMSLoose        = cms.vdouble( 0.06, 0.05, 0.05, 0.055),
-    Pt3040_RMSLoose        = cms.vdouble( 0.06, 0.05, 0.05, 0.055),
-    Pt4050_RMSLoose        = cms.vdouble( 0.06, 0.05, 0.05, 0.055)
-    )
-
```

Comment:
> > @slava77 I have tested the changes, they will work. But can you first confirm about decision above about the `cutbased` or suggest something else.
> 
> I'm not sure what `cutbased` is referring to.
> Please add a note inline in the code.

it was in pending state, I thought it was done.

Reply:
if it's still used, it's fine to keep it. However, if the ID values computed with the `PuJetIdCutBased_wp` are all long since obsolete, it may be a good time to remove the cutbased from the list

Reply:
Okay, I have commit ready then.